src/routers/features/cgn/merchants.ts

```diff
@@ -129,6 +129,15 @@ addHandler(
     ].flatMap(item => item.productCategories);
     const categoriesSet = new Set(categories);
 
+    if (req.query.count_new_discounts) {
```

Comment:
```suggestion
    if (BooleanFromString.decode(req.query.count_new_discounts).getOrElse(false)) {
```
I guess `count_new_discounts` will be always a string, since it is the query string
Does it make sense?